### PR TITLE
splash added to ios and android

### PR DESCRIPTION
**Pull request by inteljudajocu**
#### Description:

Added splash screen in ios and android

### CODE REVIEW

File: ios/COVIDSafePaths/Images.xcassets/bgImage.imageset/Contents.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
     }
   ],
   "info" : {
-    "author" : "xcode",
-    "version" : 1
+    "version" : 1,
+    "author" : "xcode"
   }
-}
+}
```
Comment (JoseFcoEstrella) [inline ios/COVIDSafePaths/Images.xcassets/bgImage.imageset/Contents.json:23]: Without line break

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM. just add line break in contents.js

**Comment by iocedano** [comment]
Could you fix the checkers that are failing?

File: ios/COVIDSafePaths/Images.xcassets/bgImage.imageset/Contents.json (hunk 1 above)
Comment (inteljudajocu) [reply ios/COVIDSafePaths/Images.xcassets/bgImage.imageset/Contents.json:23]: There is no line break

**Review by yunior05 — APPROVED** [review]
Looks good, just check the yarn errors in the CI test

**Pull request merged**